src/article/components/PostComment.jsx

```diff
@@ -0,0 +1,65 @@
+import { useContext, useState, useEffect } from "react";
+
+import IconAdd from "../../assets/icon-add.png";
+
+import { useHttpClient } from "../../hooks/http-hook";
```

Comment:
nice use of custom hooks! axios would be a good standardised option to refactor to use for handling http requests with the benefit of being fully tested but up to you - just an opinion